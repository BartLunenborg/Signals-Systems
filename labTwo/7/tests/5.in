0.6
10: [0,0,0,0,6,3,2,0,6,5]
200: [15,7,18,12,9,1,14,9,0,0,10,5,4,12,16,9,16,13,18,7,7,10,15,0,11,6,10,3,13,18,17,9,5,15,1,6,8,7,15,8,19,17,5,16,10,14,5,18,7,15,17,6,6,4,6,9,10,9,12,3,19,9,4,4,16,5,2,4,5,10,5,4,7,10,0,9,4,18,7,3,13,16,2,11,0,8,12,2,17,5,6,16,14,10,12,11,8,15,7,5,5,12,9,4,15,2,14,11,0,13,15,5,10,17,9,10,17,1,13,7,6,11,3,13,13,8,16,13,15,3,18,12,8,8,16,15,10,10,6,2,16,13,19,6,2,8,16,0,10,1,7,8,4,2,1,18,2,17,11,17,13,10,9,13,10,18,8,12,8,14,14,16,8,13,2,10,14,11,2,4,12,9,4,9,4,6,7,6,15,10,4,0,12,5,5,2,3,13,14,4]
