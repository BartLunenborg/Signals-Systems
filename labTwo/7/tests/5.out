52 0.73019
122 0.67831
151 0.68957
161 0.72636
184 0.71343
